lib/jsdom/living/custom-elements/ElementInternals-impl.js

```diff
@@ -1,5 +1,9 @@
 "use strict";
 
+// TODO: when to use generated and when to use impl directly?
```

Comment:
Generally, `require()` the generated files, as they expose the full utility APIs for each Web IDL interface.